### PR TITLE
Added recipe for EbookLib

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

File: recipes/ebooklib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - six
+
+test:
+  imports:
+    - ebooklib
+    - ebooklib.plugins
+
+about:
+  home: https://github.com/aerkalov/ebooklib
+  license: AGPL
```
Comment (ocefpaf) [inline recipes/ebooklib/meta.yaml]: AGPL-3.0 ?

File: recipes/ebooklib/setup.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git ebooklib/setup.py ebooklib/setup.py
+index b3a9a6d..37a25e8 100644
+--- ebooklib/setup.py
++++ ebooklib/setup.py
+@@ -10,7 +10,7 @@
+     url = 'https://github.com/aerkalov/ebooklib',
+     license = 'GNU Affero General Public License',
+     description = 'Ebook library which can handle EPUB2/EPUB3 and Kindle format',
+-    long_description = open('README.md').read(),
++    long_description = 'Windows and py3k had trouble with ingesting README.md here. See README.md directly',
```
Comment (ocefpaf) [inline recipes/ebooklib/setup.patch]: If you open an issue upstream link it to a feedstock issue so we can track this and remove the patch.

Comment (pmlandwehr) [inline recipes/ebooklib/setup.patch]: Turns out this was already fixed in the original repo but not yet merged into a new version for pypi - see [issue 99](https://github.com/aerkalov/ebooklib/issues/99) and [issue 104.](https://github.com/aerkalov/ebooklib/issues/104)

**Comment by pmlandwehr** [comment]
The charmap patches don't seem to have worked, so I'm reverting to my original patch - though I *will* keep the latest patches from the github repo because they solve other issues.

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

File: recipes/ebooklib/setup.patch (hunk 2 above)
Comment (jakirkham) [inline recipes/ebooklib/setup.patch]: We could also apply a patch based on commit ( https://github.com/aerkalov/ebooklib/commit/c179d7ad325639424dc19194df2e41753793d250 ). Though that, of course, would be done in the feedstock at this point.

**Pull request merged**